drivers/bus/acpi/cmbatt/cmbatt.c

```diff
@@ -1410,7 +1417,7 @@ CmBattQueryInformation(IN PCMBATT_DEVICE_EXTENSION FdoExtension,
                 else
                 {
                     /* We have data, but is it valid? */
-                    if (RemainingCapacity > 0x123456)
+                    if (RemainingCapacity > CMBATT_TIME_BOGUS)
```

Comment:
What exactly is going on with the need for this arbitrary "bogus" threshold?
Did you encounter some ACPI hardware where the reported remaining battery capacity was not one of these specific values (not zero, not "unknown" i.e. 0x7FFFFFFF), but something that looked at 1st sight legit, BUT actually was not?

Also, how did you choose this "bogus" threshold 0x123456 ? Why not e.g. 0x100000 or something else?

Reply:
I've seen some pretty wild numbers in the wild. Not trusting vendors to do the right things is a good move here IMO.
0x100000 would look prettier i guess though.

Reply:
OK then either keep the 0x123456 or 0x100000; maybe add/clarify a small comment saying (where you define the define) it's an _arbitrary threshold_ value to detect such bogus value.